beamer/health/check.py

```diff
@@ -43,7 +43,8 @@ class NotificationTypes:
 
 class HealthConfig(TypedDict):
     agent_address: str
-    deployment_dir: Path
+    artifact_dir: Path
```

Comment:
Nitpick: I think this should be called `artifacts_dir` to be more consistent, similar to above.